### PR TITLE
chore: add the job to create a pr to update the version of Cargo.toml

### PR DESCRIPTION
**Pull request by rabe1028**
## Why

- Currently, We need to create the PR to update the version in Cargo.toml
- To reduce the operation, we want to automate it.

## What

- I've added the new job to create PR to update the version in Cargo.toml
  - To run only when a release is possible, the job is run after the Release job.

## Ref

- https://github.com/nodecross/nodecross/issues/17

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -2,6 +2,15 @@ name: Release
 
 on:
   workflow_dispatch:
+    inputs:
```
Comment (miseyu) [inline .github/workflows/release.yml:5]: [Question]
Is it possible to tell at this stage whether it is major, minor, or patch?

Comment (rabe1028) [reply .github/workflows/release.yml:5]: After discussion, it was decided to do only the Bump Version before Release.
So, we will rebuild it.

**Review by daichitominaga — APPROVED** [review]
LGTM!

**Review by yokishava — APPROVED** [review]
LGTM!

**Pull request merged**